### PR TITLE
Provide cached Python package version tuple

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: thoth/common/config/runtime_environment.py [hunk 1]
```diff
@@ -44,6 +45,7 @@ class RuntimeEnvironment:
     cuda_version = attr.ib(type=str, default=None)
     name = attr.ib(type=str, default=None)
     platform = attr.ib(type=str, default=None)
+    _python_version_tuple = attr.ib(type=Optional[Tuple[int, int]], default=None)
```
Comment (KPostOffice) [inline thoth/common/config/runtime_environment.py:48]: `init=False`?

**Comment by goern** [comment]
ping?!

**Pull request merged**